TestServer_StreamTask is flaky

```
--- FAIL: TestServer_StreamTask (0.03s)
	server_test.go:312: unexpected status code: code=404, body={
		    "Error": "Not Found"
		}
```

I have been trying to track this one down but no luck. It seems that on occasion the task just doesn't get enabled but returns no errors.

--- a/cmd/kapacitord/run/server_helper_test.go
+++ b/cmd/kapacitord/run/server_helper_test.go
@@ -100,12 +100,12 @@ func (s *Server) HTTPGet(url string) (results string, err error) {
 
 func (s *Server) HTTPGetRetry(url, retry, exp string, retries int, sleep time.Duration) error {
 	var r string
-	var err error
 	for retries > 0 {
-		r, err = s.HTTPGet(url)
+		resp, err := http.Get(url)
 		if err != nil {
 			return err
 		}
+		r = string(MustReadAll(resp.Body))
 		if r == exp {
 			break
 		} else if retry == "" || r == retry {

--- a/cmd/kapacitord/run/server_test.go
+++ b/cmd/kapacitord/run/server_test.go
@@ -313,7 +313,7 @@ stream
 
 	// Request data before any writes and expect null responses
 	nullResponse := `{"Series":null,"Err":null}`
-	err = s.HTTPGetRetry(endpoint, "", nullResponse, 1000, time.Millisecond*5)
+	err = s.HTTPGetRetry(endpoint, "", nullResponse, 100, time.Millisecond*5)
 	if err != nil {
 		t.Error(err)
 	}
@@ -814,7 +814,7 @@ stream
 
 	// Request data before any writes and expect null responses
 	nullResponse := `{"Series":null,"Err":null}`
-	err = s.HTTPGetRetry(endpoint, "", nullResponse, 1000, time.Millisecond*5)
+	err = s.HTTPGetRetry(endpoint, "", nullResponse, 100, time.Millisecond*5)
 	if err != nil {
 		t.Error(err)
 	}

